docs/flow/modin/backends/pandas/parsers.rst

```diff
@@ -1,15 +1,17 @@
+:orphan:
+
 Pandas Parsers Module Description
 """""""""""""""""""""""""""""""""
 High-Level Module Overview
 ''''''''''''''''''''''''''
 
-This module houses parser classes (classes that are used for data parsing on the workers) and util functions for handling parsing results. `PandasParser` is base class for parser classes with Pandas backend, that contains methods common for all child classes. Other module classes implement `parse` function that performs parsing of specific format data basing on the chunk information computed in the `modin.engines.base.io` module. After chunk data parsing is completed, resulting `DataFrame`-s will be splitted into smaller `DataFrames` according to `num_splits` parameter, data type and number or rows/columns in the parsed chunk, and then these frames and some additional metadata will be returned.
+This module houses parser classes (classes that are used for data parsing on the workers) and util functions for handling parsing results. `PandasParser` is base class for parser classes with Pandas backend, that contains methods common for all child classes. Other module classes implement `parse` function that performs parsing of specific format data basing on the chunk information computed in the `modin.engines.base.io` module. After chunk data parsing is completed, resulting `DataFrame`-s will be splitted into smaller `DataFrames`-s according to `num_splits` parameter, data type and number or rows/columns in the parsed chunk, and then these frames and some additional metadata will be returned.
 
 Data Parsing Mechanism
 ''''''''''''''''''''''
 
 Data parsing mechanism differs depending on the data format type:
 
-* text format type - file parsing begins from retrieving `start` and `end` parameters from `parse` kwargs - these parameters define start and end bytes of data file, that should be read in the concrete partition. Using this data and file handle got from `fname`, binary data is read by python `read` function. Then resulting data is passed into `pandas.read_*` function as `io.BytesIO` object to get corresponding `pandas.DataFrame` (we need to do this because Modin partitions internally stores data as `pandas.DataFrame`).
+* text format type - file parsing begins from retrieving `start` and `end` parameters from `parse` kwargs - these parameters define start and end bytes of data file, that should be read in the concrete partition. Using this data and file handle got from `fname`, binary data is read by python `read` function. Then resulting data is passed into `pandas.read_*` function as `io.BytesIO` object to get corresponding `pandas.DataFrame` (we need to do this because Modin partitions internally stores data as `pandas.DataFrame`). TODO: add link to internal data storage implementation docs after DOCS-#2954 is merged
```

Comment:
Please use [rst commenting](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#comments), do not pollute generated docs with `TODO`-s

Reply:
Corrected.